docs/docs-beta/docs/guides/data-modeling/data-assets.md

```diff
@@ -1,4 +1,84 @@
 ---
-title: "Create data assets"
+title: "Defining data assets with decorators"
 sidebar_position: 10
 ---
+
+The most common way to create a data asset in Dagster is by annotating a function with an asset decorator. The function computes the contents of the asset, such as a database table or file.
+
+Dagster supports several ways of creating assets, but this guide will focus on using Python decorators to define data assets.
+
+## What you'll learn
+
+- The asset decorators Dagster supports
+- How to define a single asset
+- How to define operations that create multiple assets
+- How to define multiple operations that create a single asset
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- To have Dagster installed. Refer to the [Installation](/tutorial/installation) guide for more information.
+
+</details>
+
+## Asset decorators
+
+Dagster has four types of asset decorators:
+
+| Decorator | Description |
+|--------|---|
+| `@asset` | Defines a single asset. [See an example](#single-asset). |
+| `@multi_asset` | Outputs multiple assets from a single operation. [See an example](#multi-asset). |
+| `@graph_asset` | Outputs a single asset from multiple operations without making each operation itself an asset. [See an example](#graph-asset). |
+| `@graph_multi_asset` | Outputs multiple assets from multiple operations |
+
+## Defining operations that create a single asset \{#single-asset}
+
+The simplest way to define a data asset in Dagster is by using the `@asset` decorator. This decorator marks a Python function as an asset, with the function's output being the data produced by the asset.
```

Comment:
```suggestion
The simplest way to define a data asset in Dagster is by using the `@asset` decorator. This decorator marks a Python function as an asset. 
```

Reply:
@cmpadden Not sure how to best describe the role of an asset? Computing and persisting data?

Reply:
Assets represent objects in storage, for example database tables, files, or any other object produced by a data pipeline.

In Dagster when you define an asset, you write the process / compute that produces this asset / data.

Hopefully that verbiage helps spark something.